src/modules/registrypreview/RegistryPreviewUI/MainWindow.xaml

```diff
@@ -219,18 +199,13 @@
                 CornerRadius="{StaticResource OverlayCornerRadius}"
                 />
 
-            <Grid
-                Grid.Row="1"
+            <Border Style="{StaticResource GridCardStyle}"
```

Comment:
Why this change to using `Border`? `Grid` is a layout control which is more appropriate here - it also provides more children to be part of the `Grid` whereas a `Border` can only have one.

`Border` are generally used as a decorator control (to.. add a border to things :))

Reply:
Because this element only hold one child element (here: the Treeview) so there's no need for a Grid. I would've set the GridCardStyle directly on the treeview, but that caused issues (most important: the cornerradius doesn't apply). Therefore the border.